### DISCUSSION TITLE
i18n Namespace

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#2967] [Q&A] [answered]
What is 3.0 approach to registering an i18n namespace in the iModelApp? 
in 2.1x we would simply do the following
IModelApp.i18n.registerNamespace("Namespace-Name");

**Comment by pmconne** [comment, marked as answer]
`IModelApp.i18n` [has been replaced by](https://github.com/iTwin/itwinjs-core/blob/master/docs/changehistory/NextVersion.md#imodelappstartup) `IModelApp.localization`.
So replace your call with `IModelApp.localization.registerNamespace("Namespace-Name");`.

Until 3.0 officially releases with updated documentation and changelogs, please consult [NextVersion.md](https://github.com/iTwin/itwinjs-core/blob/master/docs/changehistory/NextVersion.md).
